.github/workflows/CI_merge.yml

```diff
@@ -30,11 +30,8 @@ jobs:
       - name: Unit Test
         run: npm run test:unit
 
-      - name: Test secret
-        run: |
-          [[ -z "${GH_USER_TOKEN}" ]] && echo "Exists" || echo "Does not exist"
-
-      - name: Publish
-        uses: JS-DevTools/npm-publish@v1
-        with:
-          token: ${{secrets.NPM_TOKEN}}
+      - name: Release
+        env:
+          GH_TOKEN: ${{secrets.GH_USER_TOKEN}}
+          DEBUG: semantic-release:git
```

Comment:
This needs NPM token e.g. https://github.com/genesislcap/gpl-ui/blob/master/.github/workflows/build.yml#L70

Reply:
Did you mean `NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}`?

Reply:
Yeah, link was to show the name of env var semantic release expects - you'd need to adjust the secret used as it's a different github org